### PR TITLE
fixes #12893 - ensure constants are set only once

### PR DESCRIPTION
**Pull request by ohadlevy**
the root cause is probably the same file is loaded multiple times.

### CODE REVIEW

**Comment by dLobatog** [comment]
:+1: - waiting for Jenkins to finish

File: lib/foreman/renderer.rb [hunk 1]
```diff
@@ -2,11 +2,11 @@
 
 module Foreman
   module Renderer
-    ALLOWED_HELPERS = [ :foreman_url, :grub_pass, :snippet, :snippets,
+    ALLOWED_HELPERS ||= [ :foreman_url, :grub_pass, :snippet, :snippets,
                         :snippet_if_exists, :ks_console, :root_pass,
```
Comment (dLobatog) [inline lib/foreman/renderer.rb]: 6,7,10,11 lines are not aligned after this change, rubocop failed.

**Comment by dLobatog** [comment]
@ohadlevy See comment inline, rubocop failed. Should be very easy to fix :smile:

**Comment by ohadlevy** [comment]
would you mind to fix on merge?  I wont get to it in the next hours.

**Comment by ohadlevy** [comment]
@dLobatog updated whitespaces.

**Comment by dLobatog** [comment]
Merged as c7521b7b9e4e39d526a85d162b6c1e593c2dcdda, thanks @ohadlevy!

**Pull request closed without merging**